data/templates/firewall/nftables-defines.j2

```diff
@@ -1,7 +1,7 @@
-{% macro groups(group, is_ipv6) %}
+{% macro groups(group, is_ipv6, is_l3) %}
```

Comment:
This could be defaulted to True with `is_l3=True` so it only needs defining on the bridge template.